### ISSUE TITLE
Interactive Tutorial - Creating a Cluster

### ISSUE THREAD

**Issue by sshiv5768** [kubernetes/website#33864]
Lab environment is lagging. Whenever i try to click on any steps it doesn't work.

![Screenshot from 2022-05-21 08-59-45](https://user-images.githubusercontent.com/40575397/169633788-d42a264c-cf57-4424-96be-81adefc0907a.png)

**Comment by sftim** [comment]
/label team/katacoda

Thanks for reporting this.

**Comment by Tej-Singh-Rana** [comment]
Seems like the lab environment is not loaded successfully. In the shared screenshot, I am not able to the terminal prompt. 
Could you please try to reload again? 

<img width="735" alt="katacoda-img-1" src="https://user-images.githubusercontent.com/58101587/169708103-218a701b-5799-4a3b-a194-bd2f1798b627.png">

Regards,

**Comment by sshiv5768** [comment]
Yeah, I finally found the issue. It was with network connectivity so lab was not loaded properly.
